esterni/ManualeUtente/res/sections/apputenti.tex

```diff
@@ -0,0 +1,131 @@
+	\item \textbf{Guida:} Il dipendente può visualizzare la guida premendo sull'elemento della lista "Guida" del menù principale in alto a destra; \\
+	\item \textbf{Prenota postazione:} Il dipendente può prenotare una postazione premendo sull'elemento della lista "Prenota postazione" del menù principale in alto a destra.
+	Dopo aver premuto dovrà inserire la data, l'ora di inizio, l'ora di fine e la stanza obbligatoriamente e in modo facoltativo anche il nome del collega.
+	Una volta premuto sul bottone "Cerca", se è stato inserito il nome del collega, visualizzerà tutte le sue prenotazioni effettuate nella stanza, nel range orario e nella data inseriti e scorrendo sotto visualizzerà tutte le postazioni di quella stanza con il loro stato e potrà decidere quale prenotare se disponibile. \\	
+\end{itemize}
+
+\subsection{Requisiti e installazione}
+	Per il corretto funzionamento l'app mobile richiede di essere installata su un dispositivo con sistema operativo Android con versione uguale o superiore alla 6.0 Marshmallow.
+	
+	\subsection{Utilizzo}
+	L'applicazione mobile viene utilizzata dagli utenti che vogliono accedere ad una postazione di una stanza dell'organizzazione. Essi hanno la possibilità di effettuare una prenotazione e di conoscere lo stato di una postazione, tramite la scansione del tag NFC associato. Inoltre per ogni occupazione viene registrato l'orario di inizio e di fine, e monitorata la presenza in tempo reale da parte dell'amministratore. Inoltre, ogni utente ha la possibilità di effettuare un'igienizzazione di una postazione precedentemente utilizzata. 
+	\\All'utente dell'applicazione vengono offerte tutte le funzionalità indicate in questa sezione.
+	\subsubsection{Login}
+	\begin{figure}[H]
+		\centering
+		\includegraphics[width=5cm]{res/images/login.png}
+		\caption{Login utente}
+	\end{figure}
+	L'utente può autenticarsi nella pagina del login inserendo il proprio username e la propria password nelle caselle di testo corrispondenti.
+	Nel caso in cui inserisca le credenziali corrette otterrà l'accesso all'applicazione e quindi alla pagina principale, altrimenti verrà visualizzato un messaggio di errore in cui verrà comunicato di riprovare il login.
+	
+	\subsubsection{Logout}
+	\begin{figure}[H]
+		\centering
+		\includegraphics[width=5cm]{res/images/menuATendina.png}
+		\caption{Logout utente}
+	\end{figure}
+	L'utente può eseguire il logout aprendo il menù a tendina in alto a destra e poi cliccando su logout. Avvenuta questa operazione l'utente si troverà nella pagina del login.
+	
+	\subsubsection{Scansione tag NFC}
+	\begin{figure}[H]
+		\centering
+		\includegraphics[width=5cm]{res/images/avvicinaSmartphone.png}
+		\caption{Scansione tagNFC}
+	\end{figure}
+	L'utente si trova nella pagina principale dell'applicazione e gli viene chiesto di effettuare una scansione del tag NFC tramite lo smartphone. 

```

Comment:
inserire in breve per ottenere cosa dalla scansione. ad esempio: "L'utente si trova nella pagina principale dell'applicazione e gli viene chiesto di effettuare una scansione del tag NFC tramite lo smartphone per ottenere tutte le informazioni sulla postazione."